docs/deployment-on-heroku.rst

```diff
@@ -94,23 +94,12 @@ Or add the DSN for your account, if you already have one:
 .. _Sentry add-on: https://elements.heroku.com/addons/sentry
 
 
-Gulp & Bootstrap compilation
+Webpack
 ++++++++++++++++++++++++++++
 
-If you've opted for a custom bootstrap build, you'll most likely need to setup
-your app to use `multiple buildpacks`_: one for Python & one for Node.js:
+If you use Webpack as frontend solution, it is recommended to use Docker to deploy your project to Heroku.
```

Comment:
Why this change? The multi-buildpack solution doesn't work? In my opinion, the Docker experience on Heroku was not super standard. That's a reason we don't really support Heroku + Docker.